packages/worker/src/api/routes/tests/realEmail.spec.js

```diff
@@ -3,6 +3,9 @@ const { EmailTemplatePurpose } = require("../../../constants")
 const nodemailer = require("nodemailer")
 const fetch = require("node-fetch")
 
+// for the real email tests give them a long time to try complete/fail
+jest.setTimeout(30000)
```

Comment:
👍

Reply:
Noticed it failed on yours and it had failed on one of mslouren's PRs as well - may as well throw a fix in!